spec/models/users_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+RSpec.describe User type: :model do
```

Comment:
You can modify this like so `RSpec.describe User, 'Associations', type: :model do`. In this case you don't need comments in the model.